src/transactions/actions.ts

```diff
@@ -184,5 +184,9 @@ export const navigateToPaymentTransferReview = (
 }
 
 export const navigateToRequestedPaymentReview = (transactionData: TransactionDataInput) => {
-  navigate(Screens.SendConfirmationLegacy, { transactionData, origin: SendOrigin.AppRequestFlow })
+  navigate(Screens.SendConfirmationLegacy, {
+    transactionData,
+    origin: SendOrigin.AppRequestFlow,
+    isFromScan: false,
```

Comment:
It looks like this is only used in a notifications saga right now, but it seems to me like navigating to a requested payment confirmation screen is something that could actually take place in response to a QR code scan.  In light of that, it might make more sense to make isFromScan a parameter of this function.